pkg/labelsfilter/filter_test.go

```diff
@@ -27,10 +25,6 @@ import (
 )
 
 // Hook up gocheck into the "go test" runner.
-func Test(t *testing.T) {
-	TestingT(t)
-}
-
```

Comment:
Why is this removed? Won't this lead to the `gocheck` tests no longer being run? I see no other place in this package where we hook up `gocheck` to `testing`.

Reply:
This was not intended.